apps/EchoServerClient/TcpEchoServer.v3

```diff
@@ -30,16 +30,6 @@ def main(args: Array<string>) {
 	strBldr.put1("Hello! Socket opened on fd %d\n", sock.fd);
```

Comment:
I usually use the variable name `buf` for `StringBuilder`--it used to be called `StringBuffer`, so the `buf` name was already pervasive.